apps/molgenis-components/src/components/forms/InputRef.vue

```diff
@@ -23,15 +23,18 @@
           :key="index"
         >
           <input
-            :id="`${id}-${row.name}`"
+            :id="`${id}-${flattenObject(getPrimaryKey(row, tableMetaData))}`"
```

Comment:
this usually is an indication that a component can be split up

Reply:
Sure, but that's out of scope for this pr